drivers/volume/portworx/portworx.go

```diff
@@ -83,6 +83,7 @@ const (
 	startDriverTimeout               = 2 * time.Minute
 	upgradeTimeout                   = 10 * time.Minute
 	upgradeRetryInterval             = 30 * time.Second
+	upgradePerNodeTimeout            = "900"
```

Comment:
a small cosmetic suggestion, can we use the `time.Duration` notation as we do above and then use `duration.Seconds()` in your command?

Reply:
fixed